src/h3lib/lib/mathExtensions.c

```diff
@@ -37,3 +37,14 @@ int _ipow(int base, int exp) {
 
     return result;
 }
+
+/**
+ * _imax returns the maximum of two integers.
+ */
+int _imax(int a, int b) {
+    if (a > b) {
+        return a;
+    } else {
+        return b;
+    }
+}
```

Comment:
This is the classic case of where you should use a macro. Unless you can move this to a header (requires use of `static inline` so not strictly C89 compliant), a macro is the better choice here:

```
#define _IMAX(a, b) (((a) > (b)) ? (a) : (b))
```